libse/SubtitleFormats/PhoenixSubtitle.cs

```diff
@@ -0,0 +1,123 @@
+        }
+
+        public override bool IsTimeBased
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        public override string Name
+        {
+            get
+            {
+                return "Phoenix Subtitle";
+            }
+        }
+
+        public override bool IsMine(List<string> lines, string fileName)
+        {
+            if (fileName != null && !fileName.EndsWith(".pjs", StringComparison.OrdinalIgnoreCase))
+            {
+                return false;
+            }
+            var sub = new Subtitle();
+            LoadSubtitle(sub, lines, fileName);
+            return sub.Paragraphs.Count > _errorCount;
+        }
+
+        public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string fileName)
+        {
+            var paragraph = new Paragraph();
+            for (int i = 0; i < lines.Count; i++)
+            {
+                string line = lines[i];
+                Match match = null;
+                if (line.Length > 9)
+                {
+                    match = RegexTimeCodes.Match(line);
+                }
+                if (match?.Success == true)
+                {
+                    try
+                    {
+                        // Decode times.
+                        paragraph.StartTime = DecodeTimeCode(match.Groups[1].Value);
+                        paragraph.EndTime = DecodeTimeCode(match.Groups[2].Value);
+
+                        // Decode text.
+                        line = line.Substring(match.Value.Length).Trim('"');
+                        paragraph.Text = string.Join(Environment.NewLine, line.Split('|'));
+
+                        subtitle.Paragraphs.Add(paragraph);
+                        paragraph = new Paragraph();
+                    }
+                    catch
+                    {
+                        _errorCount++;
+                    }
+                }
+                else
+                {
+                    _errorCount++;
+                }
+            }
+            subtitle.Renumber();
+        }
+
+        private static TimeCode DecodeTimeCode(string encodedTime)
+        {
+            // Format is timed in tenths of seconds. 2447
+            int time;
+            if (int.TryParse(encodedTime, out time))
+            {
+                return new TimeCode(time * .1 * TimeCode.BaseUnit);
+            }
+            return new TimeCode(0);
+        }
+
+        private static string EncodeTimeCode(TimeCode tc)
+        {
+            int tm = (int)Math.Round(tc.TotalMilliseconds);
+            return Convert.ToString(tm);
+        }
+
+        public override string ToText(Subtitle subtitle, string title)
+        {
+            const string writeFormat = "{0},   {1},   \"{2}\"\r\n";
+            var sb = new StringBuilder();
+            foreach (var p in subtitle.Paragraphs)
+            {
+                string text = HtmlUtil.RemoveHtmlTags(p.Text, true);
```

Comment:
From other PJS subtitle code I've seen, quotes inside a subtitle are not treated special (no escapes).

While parsing only the opening and closing quote should be removed. Missing quote at start and/or end of text could be ignored, and counted as an error.